desktop_version/src/FileSystemUtils.cpp

```diff
@@ -561,42 +561,16 @@ bool FILESYSTEM_openDirectoryEnabled()
 	return !SDL_GetHintBoolean("SteamTenfoot", SDL_FALSE);
 }
 
-#ifdef _WIN32
 bool FILESYSTEM_openDirectory(const char *dname)
 {
-	ShellExecute(NULL, "open", dname, NULL, NULL, SW_SHOWMINIMIZED);
+	char url[MAX_PATH];
+	SDL_snprintf(url, sizeof(url), "file://%s", dname);
+	if (SDL_OpenURL(url))
+	{
+		return false;
+	}
 	return true;
```

Comment:
1. It is true that `SDL_OpenURL()` returns -1 (a nonzero value) upon failure; however treating it directly like a boolean in a conditional is a bit obtuse. So, it ought to be `SDL_OpenURL(url) == -1`. However...
2. This construct can be simplified to `return SDL_OpenURL(url) == -1`. However...
3. We ought to print what `SDL_GetError()` gives us if `SDL_OpenURL()` fails.

Reply:
I considered that, but I was mainly trying to be consistent with the rest of `FileSystemUtils.cpp`. I'll change it, though.